ui/apps/platform/src/Containers/Network/Graph/Overlays/Legend.tsx

```diff
@@ -70,23 +68,48 @@ function Legend(): ReactElement {
     }
 
     return (
-        <div
+        <Card
```

Comment:
Now that I see we're using a PF Card, I wonder if, in the future we should adopt the [Expandable Card variant](https://www.patternfly.org/v4/components/card#expandable). It's less bespoke, and you get the indication that it's closed and can be clicked to open. (I'm not sure if we can have it open "up", though.)

@alanonthegit What do you think about that for the future?